collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1646,7 +1646,8 @@ private void createConstraintToast(FormIndex index, int saveStatus) {
             default:
                 return;
         }
-
+        SoftKeyboardUtils.hideSoftKeyboard(currentView);
+        getCurrentViewIfODKView().highlightWidget(index);
```

Comment:
But  I guess, that is for the case when there are more than one question widget on screen.

Reply:
Scrolling and highlighting is only for multiple questions on one screen it doesn't make sense to do that if you have only one question because then it's obvious which answer is wrong.

Reply:
Yes @grzesiek2010, scrolling obviously means no sense in this case.

Reply:
However, I thought of highlighting in case of a single question as well, but yes, we really do not need that. I'll remove that piece of code.